code/game/machinery/bots/secbot.dm

```diff
@@ -384,6 +384,22 @@ Auto Patrol: []"},
 		target = user
 		mode = BOT_HUNT
 
+/obj/machinery/bot/secbot/Crossed(atom/movable/AM)
+	if(ismob(AM) && target)
+		var/mob/living/carbon/C = AM
+		if(!istype(C) || !C || in_range(src, target))

```

Comment:
I highly doubt the range check is needed, how would you ```Cross()``` beepsky without being on top of him? (Hint: You wouldn't)